src/Test/Utilities/CommonTestBase.cs

```diff
@@ -100,7 +102,20 @@ static CommonTestBase()
             bool collectEmittedAssembly,
             bool verify);
 
-        private static readonly ImmutableArray<Emitter> s_emitters;
+        private static ImmutableArray<Emitter> s_emitters;
+
+        private static ImmutableArray<Emitter> Emitters
+        {
+            get
+            {
+                if (s_emitters.IsDefault)
+                {
+                    s_emitters = LoadEmitters();
```

Comment:
I didn't add it because executing `LoadEmitters` twice doesn't produce any errors.